src/Campaigns/Blocks/CampaignTitleBlock/render.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+use Give\Campaigns\Models\Campaign;
+use Give\Campaigns\Repositories\CampaignRepository;
+
+if (empty($attributes) || !isset($attributes['campaignId'])) {
```

Comment:
I'm not sure if we need to use `empty($attributes)` as part of the condition. It seems it would be enough to use only the second checking, something like this:

```
if (!isset($attributes['campaignId'])) {
    return;
}
```

Reply:
Resolved: 4b6df903c0da762a0ce6798fe4b6948124c3596d